custom/git-app-deployer/main.go

```diff
@@ -77,33 +78,39 @@ func getKubeConfig() (*rest.Config, error) {
 	return config, err
 }
 
-//GetData derive the sock-shop filePath and timeout
-//it derive the filePath based on sock-shop scenario(week vs resilient)
+//GetData derive the application filePath and timeout
+//it derive the filePath based on application scenario(week vs resilient)
 func GetData() (*AppVars, error) {
 
 	//Initialise the variables
 	namespace := flag.String("namespace", "", "namespace for the application")
 	filePath := flag.String("typeName", "weak", "type of the application")
 	timeout := flag.Int("timeout", 300, "timeout for application status")
 	operation := flag.String("operation", "apply", "type of operation for application")
+	labelApp := flag.String("label", "default", "type of label for application")
+
 	flag.Parse()
 
 	appVars := AppVars{
 		namespace: *namespace,
 		timeout:   *timeout,
 		operation: *operation,
+		label:     "app=" + *labelApp,
 	}
-	//sock-shop namespace having weak and resilient filePath
+	//application namespace having weak and resilient filePath
 	//loadtest namespace having load-test filePath
-	switch appVars.namespace {
-	case "loadtest":
-		appVars.filePath = "load-test.yaml"
-	case "sock-shop":
-		appVars.filePath = *filePath + "-sock-shop.yaml"
-	default:
-		return &appVars, fmt.Errorf("namespace '%v' not supported in app-deployer", appVars.namespace)
+	if appVars.namespace != "" {
+		switch appVars.namespace {
+		case "loadtest":
+			appVars.filePath = "loadtest.yaml"
+		case "sock-shop":
+			appVars.filePath = *filePath + "-sock-shop.yaml"
+		default:
```

Comment:
can we add one more case for podtato? keep same default -- it may eliminate the need of  extra if-else

Reply:
Here namespace may not be same so we need to categorise